src/tpg_port.c

```diff
@@ -168,25 +168,9 @@ static void port_setup_reta_table(uint8_t port, int nr_of_queus)
 /*****************************************************************************
  * port_init_tx_conf()
  ****************************************************************************/
-static void port_init_tx_conf(uint16_t port)
+static void port_init_tx_conf(uint16_t port __rte_unused)
 {
-#if !defined(TPG_SW_CHECKSUMMING)
-    struct rte_eth_txconf *tx_conf;
 
-    /* We need to enable UDP/TCP hw checksum because software checksum would
-     * take too many cpu cycles, SCTP checksum offload is not supported!
-     */
-    tx_conf = &port_dev_info[port].pi_dev_info.default_txconf;
-
-    if ((tx_conf->txq_flags & ETH_TXQ_FLAGS_NOXSUMUDP) ==
-        ETH_TXQ_FLAGS_NOXSUMUDP)
-        tx_conf->txq_flags &= ~ETH_TXQ_FLAGS_NOXSUMUDP;
-    if ((tx_conf->txq_flags & ETH_TXQ_FLAGS_NOXSUMTCP) ==
-        ETH_TXQ_FLAGS_NOXSUMTCP)
-        tx_conf->txq_flags &= ~ETH_TXQ_FLAGS_NOXSUMTCP;
-#else /* !defined(TPG_SW_CHECKSUMMING) */
-    RTE_SET_USED(port);
-#endif /* !defined(TPG_SW_CHECKSUMMING) */
```

Comment:
Hi @insanum ,

how did you decide to strip this part? I didn't find its replacement

Reply:
Hi @insanum ,
I went through your code change briefly and it looks good, if you could take care also at the scripts needed to install warp17 and travis configuration would be nice, if you don't know them it's fine, I can commit over your branch too

Reply:
> Hi @insanum ,
> 
> how did you decide to strip this part? I didn't find its replacement

Ah yes... if you look in `port_setup_port()` you'll now see a new default config section for `.txmode.offloads` that contains the flags.

Reply:
> Hi @insanum ,
> I went through your code change briefly and it looks good, if you could take care also at the scripts needed to install warp17 and travis configuration would be nice, if you don't know them it's fine, I can commit over your branch too

Sure thing. I'll take a look. :-)